install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -87,9 +87,9 @@ spec:
           or on (name)
           group by (name) (cluster_operator_up{job="cluster-version-operator"})
         ) == 1
-      for: 10m
+      for: 30m
       labels:
-        severity: critical
+        severity: warning
```

Comment:
I’m ok with this.  Warnings and degraded are for the morning.  Do we have an estimate of how much churn this removes in the fleet?

Reply:
Poking around in recent Telemetry, the improvement for running clusters is expected to be small but noticeable.  I expect impact during install to be larger, but haven't worked up numbers.  We should be able to scrape it out of test-case JUnit post-merge.